package.json

```diff
@@ -128,6 +128,8 @@
     "terra-application": "^1.48.0",
     "terra-disclosure-manager": "^4.9.0",
     "terra-enzyme-intl": "^3.0.0",
+    "terra-form-field": "^4.27.0",
+    "terra-form-fieldset": "^2.74.0",
```

Comment:
why do we need these dependencies ..??

Reply:
Added the dependencies as part of terra-framework-docs package.json.